### ISSUE TITLE
What is Extra used for?

### ISSUE THREAD

**Issue by platipusica** [thombashi/sqliteschema#2]
Hello thombashi San,

would it be possible to somehow keep the comments? In "Extra" maybe? What is Extra used for?

Thx

**Comment by thombashi** [comment]
@platipusica 
Hello,

`Extra` will be set if a schema includes extra keywords like `AUTOINCREMENT`.

You can keep comments since `sqliteschema 1.3.0`.
Please try to upgrade the package.

**Comment by platipusica** [comment]
Thank you for explanation.

I tried upgrading from  PPA, but have issues:
sudo add-apt-repository ppa:thombashi/ppa
sudo apt update
sudo apt install python3-sqliteschema
.

dpkg -l |grep schema
.
ii  python3-sqliteschema                        0.16.1-1ppa1                                 all          Dump table schema of a SQLite database file

In virtual Python env:
python dump.py 
.
Traceback (most recent call last):
  File "dump.py", line 1, in <module>
    import sqliteschema
ModuleNotFoundError: No module named 'sqliteschema'

So went with pip install, but then:
pip list |grep sqliteschema
sqliteschema                  1.2.1    

and this is not 1.3.0

Could you please update package on pip?

Thank you thombashi San!

**Comment by platipusica** [comment]
Also, 
I am interested to use your sqliteschema for this:
https://groups.google.com/g/jam-py/c/HXn13l-Z-Rc

Cheers

**Comment by thombashi** [comment]
I forgot to upload the package to PyPI.

You can get the `sqliteschea 1.3.0` with pip now.

**Comment by platipusica** [comment]
Fantastic! Thank you, that worked with closed issue:

            "Comment": "Very important comment"
            "Comment": "Another important comment"

Would you be interested in helping with the above link? It would be paid. Pls let me know. Thx

**Comment by thombashi** [comment]
Looks like the post has already been deleted?

I would help if I could.

**Comment by platipusica** [comment]
Hi,

yes, I deleted it. It was this:

Hi all,

is anyone interested into helping to develop the Python script? Talking atm
with Maxwell, but anyone can chip in with suggestions, or ideas.

Here is what I've found:
https://pypi.org/project/sqliteschema/

This can produce the schema output. After we have the output, we can loop
through each table and update admin.sqlite sys_itams with the table name
etc, and sys_fields for the fields details etc, taking care that all
datatypes are mapped to Jam supported types. I can provide the list with
that, there are not that many.

The simple script would cover all Western Characters scenarios. The UTF
support, ie German sqlite3 DB, with German chars like Straße in the table
name, can be added after. It's a start. Other DB's can be added after. Of
course, the UI is left for the User to fiddle with it. The idea is to have
a bare App, not the fully blown relations, etc at this stage.

If anyone is interested, I can provide my input what exactly is needed. PM
me. Do not want to bug Andrew, this is something the community might do,
with him overseeing if needed.

Pls let me know

Cheers




That is basically it. I can write more info if you find this interesting.
This solution would enable instant web App providing the Primary Key is
defined and types mapped to Jam.py supported types.

Pls let me know. Thx
